plutus-core/untyped-plutus-core/test/Transform/inlinePure3.uplc.golden

```diff
@@ -1 +1 @@
-(\b_5 -> (\x_8 y_9 -> 1 x_8) 1)
\ No newline at end of file
+(\b_5 y_8 -> 1 1)
```

Comment:
Why does this get renamed to `y_8`?

Reply:
Generally I wouldn't worry about trying to understand why things get renamed differently. We rename things a _lot_, it's very easy for the uniques to change more-or-less arbitrarily.